app/controllers/users_controller.rb

```diff
@@ -1,12 +1,10 @@
-class UsersController < ApplicationController
-  def new
-    @user = User.new
-  end
+class UsersController < Clearance::UsersController
 
   def create
-    @user = User.new(user_params)
+    @user = user_from_params
     if @user.save
-      session[:user_id] = @user.id
+      sign_in @user
+      flash[:notice] = "Sign Up successful! You rock."
       redirect_to dashboard_path
```

Comment:
You can combine `flash[:notice]` with `redirect_to`:

```ruby
redirect_to dashboard_path, "Sign up successful! You rock."
```

For more, find for "assign a flash" on http://api.rubyonrails.org/classes/ActionController/Redirecting.html#method-i-redirect_to